### PR TITLE
bork: init at 7.0.0

### PR DESCRIPTION
**Pull request by nicoonoclaste**
## Description of changes

Packaged `bork` v7.0.0, a Python build and release management tool.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by duckinator** [comment]
FYI @nbraud, Bork no longer uses Click as of v7.0.0.

**Comment by duckinator** [comment]
> Thanks!  That's what I seemed to recall... but it's still listed in [`pyproject.toml`](https://github.com/duckinator/bork/blob/main/pyproject.toml#L33)

Oh wow, I really dropped the ball wrt deps. Sorry about that. I'll look into pushing a v7.0.1 soon that doesn't pull in Click, documents or removes the dependency on pip, and documents or removes the indirect dependency on Rust.

**Comment by duckinator** [comment]
> though I'd like to see bork 7.0.1 build zipapps without calling it

My understanding is: if it has dependencies, this isn't possible. A zipapp includes the library and all dependencies &mdash; and those dependencies are installed via `pip install --target some_dir dist/yourproject-1.2.3.whl` and then `some_dir` is wrapped up in a ZIP file and a shebang line is slapped at the start.

The pip stage can only be skipped if you're making a zipapp of a project with no dependencies aside from the Python stdlib.

File: pkgs/development/python-modules/bork/default.nix [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+, fetchFromGitHub
+, pythonOlder
+
+, build
+, coloredlogs
+, packaging
+, toml
+, twine
+, wheel
+, ... }:
```
Comment (mkg20001) [inline pkgs/development/python-modules/bork/default.nix:12]: this is not necesarry in nixpkgs i believe

**Review by mkg20001 — APPROVED** [review]

**Pull request merged**